stdlib/test/collections/test_optional.mojo

```diff
@@ -39,10 +39,6 @@ def test_basic():
 
     assert_equal(1, a.value()[])
 
-    # Test invert operator
-    assert_false(~a)
-    assert_true(~b)
-
```

Comment:
I think Bool is getting an __invert__ method soon in another patch, can you rebase once that lands and keep these?

Reply:
nevermind, I will do this internally when I land this

Reply:
Ah, sorry I forgot that.

Reply:
I tried this but it actually doesn't work with implicit conversion to Bool. I'm not sure why, but I will just leave in the invert method for now.